ViewLayer/DisplayDashboardView/DisplayDashboardView.cpp

```diff
@@ -1,5 +1,21 @@
 #include "DisplayDashboardView.h"
 
+namespace
+{
+    const double SLOPE_R = -1;
+    const double Y_INT_R = 220;
+    const double SLOPE_G = 1.7;
+    const double Y_INT_G = 10;
+    const QString BLUE = "20";
+    const QString DEFAULT_SS = "QProgressBar:horizontal {\
```

Comment:
Overall these are poorly named variables. 

```
SLOPE_R -> RED_SLOPE
Y_INT_R -> RED_INITIAL
SLOPE_G -> GREEN_SLOPE
Y_INT_G -> GREEN_INITIAL
BLUE -> BLUE_INITIAL
DEFAULT_SS -> DEFAULT_STYLESHEET
```

Also this stylesheet solution is brittle. See if you can make it work with QPalette

```
    QPalette p = bar.palette();
    p.setColor(QPalette::Highlight, Qt::red);
    bar.setPalette(p);
```